packages/edge/src/commands/edge/drivers/default.ts

```diff
@@ -1,9 +1,9 @@
-import { EdgeDriverSummary } from '@smartthings/core-sdk'
+import { EdgeDriver } from '@smartthings/core-sdk'
 
-import { outputList, OutputListConfig } from '@smartthings/cli-lib'
+import { outputItemOrList, OutputItemOrListConfig, outputList, OutputListConfig } from '@smartthings/cli-lib'
```

Comment:
GitHub Actions points out that `OutputListConfig` doesn't need to be imported anymore.